package.json

```diff
@@ -51,6 +51,7 @@
     "typescript": "^4.8.2"
   },
   "dependencies": {
+    "nock": "^13.2.9",
```

Comment:
This should be put inside the dev dependency using --save-dev because [nock](https://github.com/nock/nock) is only used by the developer for testing purposes and will not be included in the bundle later.

Reply:
Apologize mas @faiq-naufal , already re-install with `--save-dev` to be inside dev dependency at commit [441caa3](https://github.com/inlivedev/inlive-js-sdk/pull/10/commits/441caa31ae532aebece7b16ab11a4053a05b8197). Kindly help to recheck again. Thank you.

<img width="391" alt="Screen Shot 2022-09-15 at 14 25 23" src="https://user-images.githubusercontent.com/102952824/190341641-769d9dbe-1edc-4d5d-9dcf-9bf8f7070a69.png">